### PR TITLE
Add writeShadows script to generate shadows from tokens

### PR DESCRIPTION
**Pull request by u12206050**

### CODE REVIEW

File: design-library/src/components/BccReact/BccReact.css [hunk 1]
```diff
@@ -13,25 +13,25 @@
...
     .bcc-react-emoji-list-item {
-        @apply flex cursor-pointer items-center justify-center rounded-full p-1 text-2xl leading-none shadow transition-all hover:scale-105;
+        @apply flex cursor-pointer items-center justify-center rounded-full p-1 text-2xl leading-none drop-shadow transition-all hover:scale-105;
         /* Default --not-selected */
-        @apply text-black bg-neutral-100;
+        @apply text-black bg-neutral-100 dark:bg-neutral-800 dark:text-white;
     }
     .bcc-react-emoji-list-item--selected {
-        @apply bg-neutral-800 text-white;
+        @apply bg-neutral-800 text-white dark:bg-polo-blue-800;
```
Comment (laurensgroeneveld) [inline design-library/src/components/BccReact/BccReact.css]: Until now we've not included dark mode for any components, pending the "official" dark mode implementation of the designers. Dark mode for other elements is implemented in the [overrides CSS file](https://github.com/bcc-code/bcc-event-app/blob/master/src/assets/components-overrides.css) in the Event app. For consistency, I feel we should probably keep to that approach for now, and carry these over when implementing dark mode for the whole library.

**Review by laurensgroeneveld — COMMENTED** [review]
Just a small comment on the dark mode, looks good otherwise 👍

File: design-library/src/components/BccReact/BccReact.css (hunk 1 above)
Comment (u12206050) [reply design-library/src/components/BccReact/BccReact.css]: Ok. Updated

**Review by laurensgroeneveld — APPROVED** [review]

**Comment by laurensgroeneveld** [comment]
Thanks 🙌

**Pull request merged**